packages/catalog-model/src/kinds/TemplateEntityV1alpha1.test.ts

```diff
@@ -48,6 +48,7 @@ describe('templateEntityV1alpha1Validator', () => {
             },
           },
         },
+        owner: 'team-a@example.com',
```

Comment:
hehe, i was not being clear probably - i meant to make these just "team-a", not as an email since emails aren't expected values here :)